src/datetimepicker.ios.js

```diff
@@ -81,7 +82,11 @@ export default class Picker extends React.Component<IOSNativeProps> {
         <RNDateTimePicker
           testID={testID}
           ref={this._picker}
-          style={styles.picker}
+          style={
```

Comment:
I think you should apply both styles if you can override the `style.picker` you can set the same `property` on `iOsPickerStyle`. With this implementation this comparison isn't necessary you can pass an array to style `style={[styles.picker, iOsPickerStyle]}` and on the line `72` you can set a default style with an empty object `iOsPickerStyle  = {}`

Reply:
What do you think? @lochness42

Reply:
My original thought was not to enforce overriding height when user just wants to use flex value. But using `height: 100%` isn't too bad so will use style array.